### ISSUE TITLE
show-toolbar,enable-borders settings arent getting reflected when running the program

### ISSUE THREAD

**Issue by ghost** [AnthonyDiGirolamo/todotxt-machine#44]
I have customized my settings so that todotxt-machine program starts with borders, wordwrap and toolbar enabled. But it is not happening so!! I have changed my colorscheme, todo-files and I can see the changes. So the config is read from the right file. Also fiddling with the code, I observed that you were passing in information regarding todos,colorschemes and keybindings to the urwidUI class but not so with the rest of the settings. Are you passing the rest of the settings in a different manner?

**Comment by esijg** [comment]
I seem to be running into the same problem on Ubuntu 17.04. Most settings from the config file are read but not the ones regarding toolbar, borders and such.

**Comment by GordianDziwis** [comment]
#42

**Comment by esijg** [comment]
@BonarBeavis not sure if this is the same as both pip and pip3 install todotxt-machine 2.2.2 which is the latest official release it seems.

**Comment by GordianDziwis** [comment]
@johannesgunnar latest commit in 2.2.2 was in 2015.

**Comment by esijg** [comment]
That is true, which is the latest official release according to the git repo.

Shouldn't the documentation reflect the latest official release or at least state that the config example is not valid for 2.2.2?

**Comment by 16** [comment]
I encountered the same issue and i finally install it directly with the git repository as source

    pip3 install git+https://github.com/AnthonyDiGirolamo/todotxt-machine.git

Maybe it's time to realease a `2.2.3` as the last stable ?
